### ISSUE TITLE
BUG: The hamburger icon is visible for all screensizes.

### ISSUE THREAD

**Issue by Abidsyed25** [Sriparno08/Openpedia#284]
## Category

- [ ] Documentation
- [ ] Resource Addition
- [ ] Codebase
- [X] User Interface
- [ ] Feature Request

## Description

In the "Learn Git & Github" category page, the hamburger icon is visible for all the screen sizes. It should be visible only for the mobile view or smaller screen sizes. I am attaching a screenshot below for reference. 

![title](https://github.com/Sriparno08/Openpedia/assets/116893970/d319af3a-1745-4f02-8565-e9970f5f2535)

**Comment by Sriparno08** [comment]
@Abidsyed25 Nice catch, but you're already assigned to another issue. Once its PR is merged, you will be considered for this issue. If anyone asks to be assigned to this issue before your PR is merged, he/she will be assigned.

**Comment by Abidsyed25** [comment]
@Sriparno08 please assign me this. I have already worked upon it.

**Comment by Sriparno08** [comment]
@Abidsyed25 Since you have already worked on this, we're assigning this issue to you. But from the next time, please start working after you are assigned to an issue. Otherwise, your hard work may not get counted.
